include/cutlass/layout/permute.h

```diff
@@ -453,7 +453,7 @@ class Tensor4DPermuteBMM0321ColumnMajor : public PermuteBase {
   CUTLASS_HOST_DEVICE
   LongIndex operator()(MatrixCoord coord) const {
 
-    Index BMM_batch_idx = blockIdx.z;
+    Index BMM_batch_idx = Index(blockIdx.z);
```

Comment:
The "Don't Repeat Yourself" principle suggests using `auto` here.

```suggestion
    auto BMM_batch_idx = Index(blockIdx.z);
```